### ISSUE TITLE
baidu.each支持object

### ISSUE THREAD

**Issue by nwind** [BaiduFE/Tangram-base#16]
不知道为何要限制在array中，这个从07年底al的时候就一直是array、object都支持的，为啥忽然去掉了。。。强烈要求加回来

**Comment by CatChen** [comment]
Yes... That was Fe library. It was named as Fe.each. I see no reason for a method under Array supporting Object.

**Comment by cnberg** [comment]
1.0 版本的时候是array object都支持的
1.1 的时候，erik同学为了让包分的更清楚，同时object支持实际很少被用到，因次取消了对object的支持，1.2 版本我加上了baidu.object.each。
